### PR TITLE
GEOMESA-2232 Update examples with new executable

### PR DESCRIPTION
**Pull request by atallahhezbor**
Signed-off-by: Atallah Hezbor <atallahhezbor@gmail.com>

### CODE REVIEW

File: geomesa-hbase/geomesa-hbase-tools/examples/ingest/avro/ingest_avro.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
 # type and converter example-xml is registered in $GEOMESA_HOME/conf/application.conf
-bin/geomesa-habse ingest -c catalog -s example-avro -C example-avro-no-header example_no_header.avro
+bin/geomesa-hbase ingest -c catalog -s example-avro -C example-avro-no-header example_no_header.avro
 
-bin/geomesa-habse ingest -c catalog -s example-avro -C example-avro-header example_with_header.avro
+bin/geomesa-hbase ingest -c catalog -s example-avro -C example-avro-header example_with_header.avro
```
Comment (elahrvivaz) [inline geomesa-hbase/geomesa-hbase-tools/examples/ingest/avro/ingest_avro.sh]: ah thanks, i noticed this this other day but then forgot about ti

**Comment by jnh5y** [comment]
This is a good start.  There are lots of other examples of 'geomesa' as a command by itself...  For instance, https://github.com/locationtech/geomesa/blob/master/geomesa-accumulo/geomesa-accumulo-tools/README.md.

I'd suggest looking for examples with 'git grep geomesa\ '.

**Comment by elahrvivaz** [comment]
we should probably just get rid of all the sub-modules readmes - i don't think they have been kept up to date

**Comment by elahrvivaz** [comment]
I opened a new ticket here to track the doc updates: https://geomesa.atlassian.net/browse/GEOMESA-2246

**Pull request merged**